src/sprites/rendered-target.js

```diff
@@ -1142,6 +1145,7 @@ class RenderedTarget extends Target {
     dispose () {
         this.runtime.changeCloneCounter(-1);
         this.runtime.stopForTarget(this);
+        this.runtime.removeExecutable(this);
```

Comment:
My only other question was.. is it possible for both calls to `this.runtime.removeExecutable(this)` to get called (e.g. this one and the one in `engine/target`), and if so, is that a bad thing or a good thing...

I haven't gotten a chance to investigate this yet, but just wanted to write it down for you in case you have any immediate thoughts on it. I'll try to look into this a bit tomorrow.

Reply:
It shouldn't be possible for them to both be called without `super.dispose();` being added here in `RenderedTarget`. If that was added and they were both called, `removeExecutable` is already written in a way to safely perform the removal if the target was already removed. (It'll check for existence, if it doesn't exist in the list, it won't try to alter the list.)